.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: ruby
 sudo: false
-script: bundle exec rake test
+script:
+  - bundle exec rake test
+  - bundle exec rake minitest
```

Comment:
You could have both goals on one line, so:
script: bundle exec rake test minitest